docs/practitioner/glossery.md

```diff
@@ -221,11 +221,9 @@ sidebar_position: 7
 ## Useful Terms
 
 * [Carbon Intensity](https://learn.greensoftware.foundation/practitioner/carbon-awareness#carbon-intensity) - Measures the amount of greenhouse gases emitted per unit of electricity produced.
-* [Embodied Carbon](https://learn.greensoftware.foundation/practitioner/hardware-efficiency#embodied-carbon) - The greenhouse gas emissions arising from the manufacturing, transportation, installation, maintenance, and disposal of building materials.
-* [Energy Proportionality](https://learn.greensoftware.foundation/practitioner/energy-efficiency#energy-proportionality) - Maximize the energy efficiency of hardware.
 * [Demand Shaping](https://learn.greensoftware.foundation/practitioner/carbon-awareness#demand-shaping) - The strategy of moving workloads to regions or times when the carbon intensity is less
 * [Greenhouse Gas protocol](https://ghgprotocol.org) - The most widely used and internationally recognized greenhouse gas accounting standard.
-* **Value chain emissions** - These are scope 3 emissions according to the GHG protocol, and the most significant source of emissions. They encompass the full range of activities needed to create a product or service, from conception to distribution.
-* **Energy proportionality** - Measures the relationship between power consumed by a computer and the rate at which useful work is done (its utilization).
-* **Static power draw** - This refers to how much electricity is drawn when a device is in an idle state. 
-* **Embodied carbon** (also known as "embedded carbon") - The amount of carbon pollution emitted during the creation and disposal of a device.
+* [Value chain emissions](https://www.cisl.cam.ac.uk/education/graduate-study/pgcerts/value-chain-defs) - These are scope 3 emissions according to the GHG protocol, and the most significant source of emissions. They encompass the full range of activities needed to create a product or service, from conception to distribution.
+* [Energy proportionality](https://learn.greensoftware.foundation/practitioner/energy-efficiency#energy-proportionality) - Measures the relationship between power consumed by a computer and the rate at which useful work is done (its utilization).
+* [Static power draw](https://learn.greensoftware.foundation/practitioner/energy-efficiency#static-power-draw) - This refers to how much electricity is drawn when a device is in an idle state. 
+* [Embodied Carbon](https://learn.greensoftware.foundation/practitioner/hardware-efficiency#embodied-carbon) (also known as "embedded carbon") - The amount of carbon pollution emitted during the creation and disposal of a device.
```

Comment:
Minor nit, and it should still resolve in most browsers, but the anchor links all redirect to "/#ANCHOR_TAG" when clicked. In the name of reducing overall calls and work for the server, we should be explicitly link to the directory in the following:

- https://learn.greensoftware.foundation/practitioner/energy-efficiency/#energy-proportionality
- https://learn.greensoftware.foundation/practitioner/energy-efficiency/#static-power-draw
- https://learn.greensoftware.foundation/practitioner/hardware-efficiency/#embodied-carbon

Reply:
I have updated these urls in the content that I am now submitting to the LF. pls handle the PR.

Reply:
I think we already are except value chain emissions (which we don't have a direct link to gsf website)? apologies, if i have missed sth completely...

Reply:
My comment here was calling out the missing "/" in the URL string. For example, the carbon-awareness link currently links to:

https://learn.greensoftware.foundation/practitioner/carbon-awareness#carbon-intensity

Note there is no "/" between the path and the anchor link: `carbon-awareness#carbon-intensity`. When you click a link like that, our site's web server redirects it to the following URL:

https://learn.greensoftware.foundation/practitioner/carbon-awareness/#carbon-intensity

Note the "/" is present at the end of the path and before the anchor link: `carbon-awareness/#carbon-intensity`. 

This is quite minor and doesn't block any functionality or links.

Reply:
guys it's all sorted in the google docs. can you update the website as well so we can close this and announce 'code freeze' :)